sdk/core/azure-core/inc/azure/core/http/winhttp/win_http_client.hpp

```diff
@@ -7,8 +7,6 @@
 
 #pragma once
 
-#if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
```

Comment:
I am curious what's the benefit of removing this from the header file and push it onto each user who includes it (like the winhttp cpp file)? Trying to understand the motivation of this change.